.gitignore

```diff
@@ -1,3 +1,5 @@
+old_multiselect/
```

Comment:
I assume that this is just a relic of your local dev directory structure.  You should just revert these changes to the .gitignore file.